dhall-json bounds on Hackage

Currently seem to be broken: see [matrix CI]

Thanks! :)

Can you give more details?

I see https://matrix.hackage.haskell.org/#/package/dhall-json/1.6.4/ghc-8.0.2@1589854941, but the error seems very mysterious:

``` 
src/Dhall/JSONToDhall.hs:361:1: error:
    Failed to load interface for ‘Data.Map.Merge.Lazy’
    It is a member of the hidden package ‘containers-0.6.2.1’.
    Perhaps you need to add ‘containers’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

Can you reproduce that?

--- a/dhall-json/dhall-json.cabal
+++ b/dhall-json/dhall-json.cabal
@@ -43,7 +43,7 @@ Library
         aeson-pretty                             < 0.9 ,
         aeson-yaml                >= 1.0.6    && < 1.1 ,
         bytestring                               < 0.11,
-        containers                                     ,
+        containers                >= 0.5.9    && < 0.7 ,
         dhall                     >= 1.32.0   && < 1.33,
         exceptions                >= 0.8.3    && < 0.11,
         filepath                                 < 1.5 ,

